src/components/victory-line/helper-methods.js

```diff
@@ -6,6 +6,13 @@ import Scale from "../../helpers/scale";
 
 export default {
 
+  getScale(props, fallbackProps) {
+    const modifiedProps = Helpers.modifyProps(props, fallbackProps);
+    const {scale} = this.getCalculatedValues(modifiedProps);
```

Comment:
`getCalculatedValues` should rely on `getScale` not the other way around to avoid unnecessary calculations.